tests/test_driver/firebase_auth/firebase_auth_instance_e2e.dart

```diff
@@ -191,9 +191,6 @@ void setupTests() {
         await FirebaseAuth.instance.currentUser!
             .updateDisplayName('updatedName');
 
-        if (!kIsWeb) {
-          await FirebaseAuth.instance.currentUser!.reload();
-        }
```

Comment:
This line is removed so we are now consistent between Mobile and Web, but not sure if it was only a test issue?

Reply:
Seems to be good on release